resources/styles/abstracts/_variables.scss

```diff
@@ -0,0 +1,7 @@
+:root {
+  /**
```

Comment:
It seems that incorrect formatting is placed here. Should we keep comments in the code?

Reply:
I think we can keep comments in such a "config" scss files to leave information about the mixins, variables etc. What do you think?